Moving Hosted Che FAQ to the che-docs (#1295)
codenvy/che-docs: src/main/pages/che-7/overview/ref_hosted-che-faq-and-troubleshooting.adoc

// Module included in the following assemblies:
//
// hosted-{prod-id-short}
:page-liquid:

[id="hosted-che-faq-and-troubleshooting_{context}"]
= Frequently Asked Questions (FAQs) and Troubleshooting

== FAQs

**Q**: Where can I see the up-to-date status of the Eclipse Che hosted by Red Hat?

**A**: There is a dedicated status page - https://che.statuspage.io/

== Troubleshooting

> === Can not login to the Hosted Che - Authorization token is missed

Reviewer: ```suggestion
=== Cannot log into the Hosted Che - Authorization token is missed
```

To authenticate to https://che.openshift.io, one needs to allow cookies from the `static.developers.redhat.com`.
In case these cookies are blocked (by a browser extension like https://www.eff.org/privacybadger[Privacy Badger]),
authentication fails with the following error:

----

> Authorization token is missed

Reviewer: Is this the correct output? If so, we'll need to fix it in the source code. The correct phrasing would be "Authorization token is missing"

Author: what is wrong with this passive voice structure?

Reviewer: It's not grammatically incorrect, just has a slightly different meaning. 
"X is missing" means something is absent. I assume that's what the output error says.

"X is missed" means that somebody has missed X, for example "If the deadline is missed"="if somebody missed the deadline", in Russian it would be "упущено/упустил". Hope that makes sense.

Author: thanks, for clarification. Labeling is appeared to be fixed in the code https://github.com/eclipse/che/blob/ce64b1966e0a13b6da6732e9881c5215f8bd6c26/multiuser/keycloak/che-multiuser-keycloak-server/src/main/java/org/eclipse/che/multiuser/keycloak/server/KeycloakEnvironmentInitializationFilter.java#L139

Author: fixed
